src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimantResponseService.java

```diff
@@ -179,6 +202,11 @@ private boolean isStatePaid(Response response) {
             && ((FullDefenceResponse) response).getDefenceType() == DefenceType.ALREADY_PAID;
     }
 
+    private boolean isFullDefenseDisputeAcceptation(Response response, ClaimantResponse claimantResponse) {
+        return ACCEPTATION == claimantResponse.getType()
```

Comment:
I know this is the same format as the method below but it reads better to swap round the above parameters. eg
```suggestion
        return claimantResponse.getType() == ACCEPTATION
```